stubs/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.stub

```diff
@@ -8,4 +8,9 @@ abstract class KernelTestCase
      * @var TestContainer
      */
     protected static $container;
+
+    /**
+     * @return TestContainer
+     */
+    abstract public function getContainer();
```

Comment:
Also think this should be `protected static` like the implemntation: https://github.com/symfony/symfony/blob/94533d419fb54aac8c9391fa9cbe03342ee8f374/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L88